Whisky.xcodeproj/project.pbxproj

```diff
@@ -552,7 +555,7 @@
 				CURRENT_PROJECT_VERSION = 12;
 				DEVELOPMENT_ASSET_PATHS = "\"Whisky/Preview Content\"";
 				DEVELOPMENT_TEAM = "";
-				"DEVELOPMENT_TEAM[sdk=macosx*]" = 92S3SG4PTH;
+				"DEVELOPMENT_TEAM[sdk=macosx*]" = BNZ74W4Z78;
```

Comment:
Make sure you don't commit this part of the `pbxproj` file

Reply:
I've changed the ID back on my fork, not sure if the PR is correctly reflecting this though